src/napi/napi.zig

```diff
@@ -832,7 +832,7 @@ pub export fn napi_get_typedarray_info(
     maybe_length: ?*usize,
     maybe_data: ?*[*]u8,
     maybe_arraybuffer: ?*napi_value,
-    maybe_byte_offset: ?*usize,
+    maybe_byte_offset: ?*usize, // TODO: remove this? it's always 0
```

Comment:
for compatibility with Node-API we need to keep these args, but yeah doesn't make much sense to have it